releasenotes/notes/matplotlib_deprecate_fix_11144-914a20fc305d5432.yaml

```
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    Instead use `draw` method in `~qiskit..visualization.pulse_v2.interface.py`
+deprecations:
+  - |
+    The deprecated function in :func:`~qiskit..visualization.pulse.matplotlib.py` will be remove in the future release
+fixes:
+  - |
+    Issue #11144 `Remove deprecated code since 0.23`
```

```suggestion
upgrade:
  - |
     :func:`qiskit.visualization.pulse.matplotlib.py` is deprecated with version 0.23, instead   
     use :func `qiskit.visualization.pulse_drawer`.

```
Also to link an issue use fixes or closes or fix etc. It fixes #11144